### PR TITLE
Updated api to now get dok and math_practice denorm fields.

### PR DESCRIPTION
**Pull request by bgsavage**
Some renames of different item fields for consistency as they are pointing to codes.  Renamed interpretiveGuide -> ...Url

@agorina  after this merge the "normalized" cols for these tables can be candidates for removal.

### CODE REVIEW

File: webapp/src/main/java/org/opentestsystem/rdw/reporting/exam/AssessmentItem.java [hunk 1]
```diff
@@ -11,13 +11,13 @@
     private long id;
     private int position;
     private String bankItemKey;
-    private String claim;
-    private String target;
-    private String depthOfKnowledge;
+    private String claimCode;
+    private String targetCode;
+    private String depthOfKnowledgeCode;
```
Comment (bgsavage) [inline webapp/src/main/java/org/opentestsystem/rdw/reporting/exam/AssessmentItem.java]: @esotrope  Updated for consistency

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 2]
```diff
@@ -523,35 +523,35 @@
       }
     },
     "dok": {
-      "1": {
+      "1_1": {
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: Now using code in en.json

File: webapp/src/test/resources/integration-test-data.sql [hunk 3]
```diff
@@ -59,10 +59,10 @@ insert into math_practice (practice, description, code) values
...
   (-1, 1, 1, 'dok1', 'dok1');
 
-insert into item (id, natural_id, claim_id, claim_code, target_id, target_code, asmt_id, math_practice, dok_id, dok_level_subject_id, difficulty_code, max_points, common_core_standard_ids) values
-  (-1, '200-3391', -1, 'c1', -1, 't1', -1, -1, -1, '-1_1', 'E', 3, 'S-ID.1'),
-  (-2, '200-3392', -1, 'c1', -2, 't2', -1, -1, -1, '-1_1', 'D', 4, 'S-ID.3'),
-  (-3, '200-3393', -1, 'c1', -3, 't3', -1, -1, -1, '-1_1', 'M', 5, 'S-ID.4|S-ID.7');
+insert into item (id, natural_id, claim_id, claim_code, target_id, target_code, asmt_id, math_practice, math_practice_code, dok_id, dok_level_subject_id, difficulty_code, max_points, common_core_standard_ids) values
+  (-1, '200-3391', -1, 'c1', -1, 't1', -1, -1, '-1', -1,'-1_1', 'E', 3, 'S-ID.1'),
+  (-2, '200-3392', -1, 'c1', -2, 't2', -1, -1, '-1', -1, '-1_1', 'D', 4, 'S-ID.3'),
+  (-3, '200-3393', -1, 'c1', -3, 't3', -1, -1, '-1', -1, '-1_1', 'M', 5, 'S-ID.4|S-ID.7');
```
Comment (bgsavage) [inline webapp/src/test/resources/integration-test-data.sql]: Added math_practice_code to IT

File: webapp/src/main/java/org/opentestsystem/rdw/reporting/exam/AssessmentItem.java (hunk 1 above)
Comment (esotrope) [inline webapp/src/main/java/org/opentestsystem/rdw/reporting/exam/AssessmentItem.java]: thanks!

**Pull request merged**